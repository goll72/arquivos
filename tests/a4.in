5 ataque14.bin 1
9933 2017 82.93 "GERMANY" "PHISHING" NULO "VPN"
