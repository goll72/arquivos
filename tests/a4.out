175158.060000
